docs/reference2/sql-syntax/data-types/compound.md

```diff
@@ -21,8 +21,8 @@ ksqlDB supports fields that are arrays of another type. All of the elements
 in the array must be of the same type. The element type can be any valid
 SQL type.
 
-The elements of an array are zero-indexed and can be accessed by using
-the `[]` operator passing in the index. For example, `SOME_ARRAY[0]`
+The elements of an array are one-indexed and can be accessed by using
```

Comment:
I had to find the code to prove to myself this is true: https://github.com/confluentinc/ksql/blob/master/ksqldb-execution/src/main/java/io/confluent/ksql/execution/codegen/helpers/ArrayAccess.java.

Reply:
Yes, `1` is right. I copied in some older docs here.